components/Navbar.js

```diff
@@ -10,86 +10,80 @@ const Navbar = () => {
   const { publicKey } = useWallet();
 
   return (
-    <nav className="bg-orange-400 px-2 sm:px-4 py-2.5">
-      <div className="container flex flex-wrap items-center justify-between mx-auto">
-        <a href="https://flowbite.com/" class="flex items-center">
-          <span className="self-center text-xl font-semibold whitespace-nowrap dark:text-white">
-            BONKETY BONK
+    <nav class="bg-white px-2 sm:px-4 py-2.5 dark:bg-gray-900 fixed w-full z-20 top-0 left-0 border-b border-gray-200 dark:border-gray-600">
+      <div class="container flex flex-wrap items-center justify-between mx-auto">
+        <a href="#" class="flex items-center">
+          <span class="self-center text-xl font-semibold whitespace-nowrap text-white">
+            BonketyBonk
           </span>
         </a>
-        <button
-          data-collapse-toggle="navbar-default"
-          type="button"
-          className="inline-flex items-center p-2 ml-3 text-sm text-gray-500 rounded-lg md:hidden hover:bg-gray-100 focus:outline-none focus:ring-2 focus:ring-gray-200 dark:text-gray-400 dark:hover:bg-orange-700 dark:focus:ring-gray-600"
-          aria-controls="navbar-default"
-          aria-expanded="false"
-        >
-          <span class="sr-only">Open main menu</span>
-          <svg
-            className="w-6 h-6"
-            aria-hidden="true"
-            fill="currentColor"
-            viewBox="0 0 20 20"
-            xmlns="http://www.w3.org/2000/svg"
+        <div class="flex md:order-2">
+          <button
```

Comment:
We shouldnt use a button here otherwise it renders a button inside of a button. Instead we can change the design in the global.css file by using the `wallet-adapter-button` class. Can also use tailwind there using @apply